fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -4079,6 +4075,14 @@ Reference<TCTeamInfo> DDTeamCollection::buildLargeTeam(int teamSize) {
 		}
 	}
 
+	if (totalShardCount >= SERVER_KNOBS->DD_MAXIMUM_LARGE_TEAMS) {
+		TraceEvent(SevWarnAlways, "TooManyLargeTeams", distributorId)
```

Comment:
"TooManyShardsOnLargeTeams" might be a more accurate trace message, though perhaps more verbose than ideal.